tests/importer/importer.cpp

```diff
@@ -1070,3 +1070,39 @@ TEST(Importer, importingCommonUnitsDefinitions)
     importer->resolveImports(model, resourcePath("importer/"));
     EXPECT_EQ(size_t(0), importer->issueCount());
 }
+
+TEST(Importer, baseFileEndingInBackSlash)
```

Comment:
The test should be called `baseFileEndingInForwardSlash`.

Reply:
2d8a05a